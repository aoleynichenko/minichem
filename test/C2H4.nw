start C2H4
memory 400 mb
echo

geometry
C       -0.034486550      0.691394960      0.000000000
C        0.028141480     -0.613093450      0.000000000
H       -0.976771620      1.223168460      0.000000000
H        0.852662750      1.310733540      0.000000000
H        0.970605920     -1.144586200      0.000000000
H       -0.858865390     -1.232589870      0.000000000
end

basis "ao basis" SPHERICAL
H    S
      3.42525091             0.15432897       
      0.62391373             0.53532814       
      0.16885540             0.44463454  
C    S
     71.6168370              0.15432897       
     13.0450960              0.53532814       
      3.5305122              0.44463454       
C    S
      2.9412494             -0.09996723
      0.6834831              0.39951283    
      0.2222899              0.70011547            
C    P
      2.9412494              0.15591627       
      0.6834831              0.60768372       
      0.2222899              0.39195739    
end

scf
  guess eht
end

out
  molden
end

task scf
