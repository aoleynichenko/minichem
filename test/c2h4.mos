[Molden Format]
[Atoms] Angs
  C   1   6   -0.06517013    1.30654702    0.00000000
  C   2   6    0.05317969   -1.15857863    0.00000000
  H   3   1   -1.84583072    2.31145323    0.00000000
  H   4   1    1.61129896    2.47692724    0.00000000
  H   5   1    1.83417923   -2.16295429    0.00000000
  H   6   1   -1.62302025   -2.32925711    0.00000000
[5D]
[GTO]
   1
 s  3
 7.16168370e+01  1.54328970e-01
 1.30450960e+01  5.35328140e-01
 3.53051220e+00  4.44634540e-01
 s  3
 2.94124940e+00  -9.99672300e-02
 6.83483100e-01  3.99512830e-01
 2.22289900e-01  7.00115470e-01
 p  3
 2.94124940e+00  1.55916270e-01
 6.83483100e-01  6.07683720e-01
 2.22289900e-01  3.91957390e-01

   2
 s  3
 7.16168370e+01  1.54328970e-01
 1.30450960e+01  5.35328140e-01
 3.53051220e+00  4.44634540e-01
 s  3
 2.94124940e+00  -9.99672300e-02
 6.83483100e-01  3.99512830e-01
 2.22289900e-01  7.00115470e-01
 p  3
 2.94124940e+00  1.55916270e-01
 6.83483100e-01  6.07683720e-01
 2.22289900e-01  3.91957390e-01

   3
 s  3
 3.42525091e+00  1.54328970e-01
 6.23913730e-01  5.35328140e-01
 1.68855400e-01  4.44634540e-01

   4
 s  3
 3.42525091e+00  1.54328970e-01
 6.23913730e-01  5.35328140e-01
 1.68855400e-01  4.44634540e-01

   5
 s  3
 3.42525091e+00  1.54328970e-01
 6.23913730e-01  5.35328140e-01
 1.68855400e-01  4.44634540e-01

   6
 s  3
 3.42525091e+00  1.54328970e-01
 6.23913730e-01  5.35328140e-01
 1.68855400e-01  4.44634540e-01

[MO]
Sym=C1
Ene=  -299.9162123
Spin=Alpha
Occup=2.00000000
   1  -0.70144346
   2  -0.01999889
   3   0.00009721
   4  -0.00204049
   5   0.00000000
   6  -0.70212352
   7  -0.02002979
   8  -0.00009774
   9   0.00203572
  10  -0.00000000
  11   0.00474690
  12   0.00474750
  13   0.00475188
  14   0.00475217
Sym=C1
Ene=  -299.8876969
Spin=Alpha
Occup=2.00000000
   1   0.70178576
   2   0.03160878
   3   0.00021691
   4  -0.00450629
   5  -0.00000000
   6  -0.70110534
   7  -0.03158937
   8   0.00021648
   9  -0.00450854
  10  -0.00000000
  11  -0.00492414
  12  -0.00492456
  13   0.00491955
  14   0.00491989
Sym=C1
Ene=   -26.8757356
Spin=Alpha
Occup=2.00000000
   1  -0.17952858
   2   0.46803903
   3   0.00574685
   4  -0.11935880
   5  -0.00000000
   6  -0.17952979
   7   0.46804343
   8  -0.00572786
   9   0.11936124
  10  -0.00000000
  11   0.11195368
  12   0.11196867
  13   0.11195980
  14   0.11196805
Sym=C1
Ene=   -20.2922908
Spin=Alpha
Occup=2.00000000
   1   0.13563651
   2  -0.41005019
   3   0.00972424
   4  -0.20311191
   5   0.00000000
   6  -0.13563553
   7   0.41004415
   8   0.00971529
   9  -0.20309071
  10   0.00000000
  11  -0.22358429
  12  -0.22359084
  13   0.22354089
  14   0.22361553
Sym=C1
Ene=   -16.4801331
Spin=Alpha
Occup=2.00000000
   1  -0.00001649
   2   0.00006677
   3  -0.39641352
   4  -0.01891171
   5   0.00000000
   6   0.00000315
   7  -0.00000467
   8  -0.39643713
   9  -0.01911546
  10   0.00000000
  11   0.25662755
  12  -0.25652409
  13  -0.25659120
  14   0.25661701
Sym=C1
Ene=   -14.7007371
Spin=Alpha
Occup=2.00000000
   1  -0.01489599
   2   0.01686279
   3   0.02384606
   4  -0.49939718
   5   0.00000000
   6  -0.01488605
   7   0.01682705
   8  -0.02404689
   9   0.49939590
  10  -0.00000000
  11  -0.21691729
  12  -0.21702912
  13  -0.21699602
  14  -0.21697314
Sym=C1
Ene=   -12.4644601
Spin=Alpha
Occup=2.00000000
   1  -0.00000518
   2   0.00001878
   3   0.39293794
   4   0.01881854
   5  -0.00000000
   6   0.00000762
   7  -0.00004573
   8  -0.39289867
   9  -0.01884994
  10  -0.00000000
  11  -0.35062471
  12   0.35063124
  13  -0.35063953
  14   0.35055780
Sym=C1
Ene=    -9.1296573
Spin=Alpha
Occup=2.00000000
   1  -0.00000000
   2  -0.00000000
   3   0.00000000
   4   0.00000000
   5   0.63195552
   6  -0.00000000
   7   0.00000000
   8  -0.00000000
   9   0.00000000
  10   0.63195908
  11  -0.00000000
  12   0.00000000
  13   0.00000000
  14  -0.00000000
Sym=C1
Ene=     8.9342086
Spin=Alpha
Occup=0.00000000
   1   0.00000000
   2  -0.00000000
   3   0.00000000
   4  -0.00000000
   5   0.81757340
   6  -0.00000000
   7   0.00000000
   8   0.00000000
   9  -0.00000000
  10  -0.81757065
  11   0.00000000
  12  -0.00000000
  13  -0.00000000
  14  -0.00000000
Sym=C1
Ene=    16.8389032
Spin=Alpha
Occup=0.00000000
   1  -0.00003462
   2   0.00020340
   3  -0.69745963
   4  -0.03343560
   5  -0.00000000
   6   0.00008978
   7  -0.00052148
   8  -0.69739981
   9  -0.03336471
  10  -0.00000000
  11  -0.62641443
  12   0.62624027
  13   0.62665452
  14  -0.62592470
Sym=C1
Ene=    19.0456670
Spin=Alpha
Occup=0.00000000
   1   0.12296441
   2  -0.82502473
   3   0.02517058
   4  -0.51994383
   5   0.00000000
   6   0.12365283
   7  -0.82935481
   8  -0.02500845
   9   0.52100374
  10  -0.00000000
  11   0.63019441
  12   0.62976129
  13   0.63258717
  14   0.63278925
Sym=C1
Ene=    19.1474786
Spin=Alpha
Occup=0.00000000
   1   0.16779453
   2  -1.05787245
   3   0.01080112
   4  -0.23083595
   5  -0.00000000
   6  -0.16728032
   7   1.05441624
   8   0.01069455
   9  -0.22863220
  10   0.00000000
  11   0.64738455
  12   0.64783159
  13  -0.64468817
  14  -0.64525023
Sym=C1
Ene=    25.7900764
Spin=Alpha
Occup=0.00000000
   1  -0.00007541
   2   0.00058883
   3  -0.95719774
   4  -0.04692446
   5  -0.00000000
   6   0.00010922
   7  -0.00086303
   8   0.95736208
   9   0.04513340
  10   0.00000000
  11  -0.61212236
  12   0.61255751
  13  -0.61241994
  14   0.61237579
Sym=C1
Ene=    27.9715574
Spin=Alpha
Occup=0.00000000
   1  -0.11584145
   2   0.93004717
   3   0.05733052
   4  -1.17800729
   5   0.00000000
   6   0.11585404
   7  -0.93013715
   8   0.05577109
   9  -1.17802549
  10  -0.00000000
  11   0.11804640
  12   0.11697236
  13  -0.11689125
  14  -0.11797505

