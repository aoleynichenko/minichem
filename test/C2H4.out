Minichem 0.0
By Alexander Oleynichenko, 2016
Build:      1281
Build date: Mar  9 2016 21:44:01
Compiler:   gcc 5.2.0
Date:       Wed Mar  9 22:44:26 2016
Cores:      1
Arguments:  ../bin/minichem C2H4.nw 
------------------------------------------------------------
>>> C2H4.nw
start C2H4
memory 400 mb
echo

geometry
C       -0.034486550      0.691394960      0.000000000
C        0.028141480     -0.613093450      0.000000000
H       -0.976771620      1.223168460      0.000000000
H        0.852662750      1.310733540      0.000000000
H        0.970605920     -1.144586200      0.000000000
H       -0.858865390     -1.232589870      0.000000000
end

# STO-3G
basis "ao basis" SPHERICAL
H    S
      3.42525091             0.15432897       
      0.62391373             0.53532814       
      0.16885540             0.44463454  
C    S
     71.6168370              0.15432897       
     13.0450960              0.53532814       
      3.5305122              0.44463454       
C    S
      2.9412494             -0.09996723
      0.6834831              0.39951283    
      0.2222899              0.70011547            
C    P
      2.9412494              0.15591627       
      0.6834831              0.60768372       
      0.2222899              0.39195739    
end

scf
  diis 4
  guess eht
end

out
  molden
end

task scf
------------------------------------------------------------
                    Basis information
   H
  ----
  1   1S     3.42525091     0.15432897
             0.62391373     0.53532814
             0.16885540     0.44463454


   C
  ----
  1   1S    71.61683700     0.15432897
            13.04509600     0.53532814
             3.53051220     0.44463454

  2   2S     2.94124940    -0.09996723
             0.68348310     0.39951283
             0.22228990     0.70011547

  3   2P     2.94124940     0.15591627
             0.68348310     0.60768372
             0.22228990     0.39195739


Spherical basis set summary:
*---------*-----------------*--------*------------*
| Element |   Description   | Shells | Primitives |
*---------*-----------------*--------*------------*
|   H     | ao basis        |   1    |     3      |
|   C     | ao basis        |   3    |     9      |
*---------*-----------------*--------*------------*

------------------------------------------------------------
          ********************************
          *      Parallel SCF Module     *
          ********************************

Formula:   
Atoms:     6
Electrons: 16
Mol mass:  28.053
Charge:    0
Spin mult: 1
Units:     angstroms
------------------------------------------------------------
One-electron integrals evaluation algorithm: Obara-Saika
Two-electron integrals evaluation algorithm: Obara-Saika
One-electron integrals done in 0.000004 sec

****************** BASIS ORTHOGONALIZATION *******************
Algorithm: canonical
Basis functions elimination threshold: 1e-08
Overlap matrix lowest eigenvalue = 0.17870619
AO basis orthogonalization done in 0.000128 sec

Initial guess: extended Huckel theory
Initial energy = -110.21938626
Initial guess done in 0.003602 sec

#bfns = 14
#eris = 5565

 iter.       Energy         Delta E       RMS-Dens       DIIS-Err     time
----------------------------------------------------------------------------
   1     -77.76137854    -1.28819813     0.52979118     0.30953427    0.13
   2     -76.93390212     0.82747642     0.12860660     0.02571844    0.27
   3     -77.09003322    -0.15613109     0.02496905     0.00202482    0.40
   4     -77.07385622     0.01617699     0.00228328     0.00031134    0.53
   5     -77.07403625    -0.00018003     0.00013815     0.00002962    0.66
   6     -77.07395243     0.00008382     0.00001880     0.00000454    0.79
   7     -77.07395536    -0.00000293     0.00000131     0.00000027    0.92
   8     -77.07395495     0.00000041     0.00000033     0.00000021    1.05
----------------------------------------------------------------------------
          Total SCF energy =   -77.07395495
  Nuclear repulsion energy =    33.74620585


      Time for:           sec
---------------------------------
  Orthogonalization      0.000
  Initial guess          0.004
  Density matrix         0.033
  Fock matrix            0.993
  Diagonalization        0.001
---------------------------------


         Molecular Orbitals Summary
  +-----+-----+----------------+----------+
  | No  | Occ |     Energy     | Symmetry |
  +-----+-----+----------------+----------+
  | 1   |  2  |   -11.02171351 |     ?    |
  | 2   |  2  |   -11.02066559 |     ?    |
  | 3   |  2  |    -0.98766471 |     ?    |
  | 4   |  2  |    -0.74572766 |     ?    |
  | 5   |  2  |    -0.60563350 |     ?    |
  | 6   |  2  |    -0.54024193 |     ?    |
  | 7   |  2  |    -0.45806029 |     ?    |
  | 8   |  2  |    -0.33550860 |     ?    |
  | 9   |  0  |     0.32832599 |     ?    |
  | 10  |  0  |     0.61881805 |     ?    |
  | 11  |  0  |     0.69991510 |     ?    |
  | 12  |  0  |     0.70365660 |     ?    |
  | 13  |  0  |     0.94776748 |     ?    |
  | 14  |  0  |     1.02793540 |     ?    |
  +-----+-----+----------------+----------+

        Mulliken population analysis
        ----------------------------

      Atom     Population     Charge
      1  C     6.12712763  -0.12712763
      2  C     6.12713561  -0.12713561
      3  H     0.93643658   0.06356342
      4  H     0.93643728   0.06356272
      5  H     0.93643774   0.06356226
      6  H     0.93642516   0.06357484


        Loewdin population analysis
        ---------------------------

      Atom     Population     Charge
      1  C     6.06585227  -0.06585227
      2  C     6.06586013  -0.06586013
      3  H     0.96707278   0.03292722
      4  H     0.96707592   0.03292408
      5  H     0.96707289   0.03292711
      6  H     0.96706602   0.03293398

               Multipole moments
               -----------------
  D       x            y            z
     0.00000831   0.00000596   0.00000000

  |D| = 0.00001023 a.u. = 0.00002599 Debye


Allocated: 127488 bytes
Freed:     114544 bytes
Max usage: 30976 bytes
Total time: 1.06 sec
